src/main/java/org/zone/commands/structure/misc/Messages.java

```diff
@@ -0,0 +1,299 @@
+        return Component.text("A region needs to be started. Use /zone create bounds " + "<name...>")
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunopParentEmptyError(Zone zone) {
+        return Component.text("Could not find parent zone of " + zone.getParentId().orElse("None")).color(NamedTextColor.RED);
+    }
+
+    public static Component getZonesCreateEndCommandrunZoneCreated(@SuppressWarnings("TypeMayBeWeakened") Zone zone) {
+        return Component
+                .text("Created a new zone of ")
+                .append(Component
+                                .text(zone.getName())
+                                .color(NamedTextColor.AQUA));
+    }
+
+    public static Component getCommandLauncherrunMethodError1Else() {
+        return Component.text("Unknown error").color(NamedTextColor.RED);
+    }
+
+    public static Component getCommandLauncherrunMethodNoPermissionError(ArgumentCommand argumentCommand) {
+        return Component
+                .text("You do not have permission for that command. You " +
+                              "require " +
+                              argumentCommand.getPermissionNode())
+                .color(NamedTextColor.RED);
+    }
+
+    public static Component getDefaultFlagFileFlagLoadFail(@SuppressWarnings("TypeMayBeWeakened") FlagType<?> type) {
+        return Component
+                .text("Failed to load flag of " + type.getId())
+                .color(NamedTextColor.RED)
+                .decorate(TextDecoration.BOLD);
+    }
+
+    public static Component getZoneFlagMemberGroupViewCommandrunPagelessthan1() {
+        return Component.text("Page needs to be 1 or more").color(NamedTextColor.RED);
+    }
+
+    public static Component getZoneFlagMemberGroupViewCommand(@SuppressWarnings("TypeMayBeWeakened") Group group) {
+        return Component
+                .text("Group: ")
+                .color(NamedTextColor.GOLD)
+                .append(Component
+                                .text(group.getName())
+                                .color(NamedTextColor.AQUA));
+    }
+
+    public static Component getZoneFlagMemberGroupViewCommandTotal(Collection<UUID> memberIds) {
+        return Component
+                .text("Total: ")
+                .color(NamedTextColor.GOLD)
+                .append(Component
+                                .text(memberIds.size())
+                                .color(NamedTextColor.AQUA));
+    }
+
+    public static Component getZoneFlagMemberGroupViewCommandPages(int page) {
+        return Component
+                .text("Page: ")
+                .color(NamedTextColor.GOLD)
+                .append(Component.text(page).color(NamedTextColor.AQUA));
+    }
+
+    public static Component getZoneInfoCommandZoneName(@SuppressWarnings("TypeMayBeWeakened") Zone zone) {
+        return Component.text("Name: " + zone.getName()).color(NamedTextColor.AQUA);
```

Comment:
This should probably call PluginInfo and be weakened

Reply:
emm ok

Reply:
@mosemister err wdym PluginInfo? why it's not the Plugin info command

Reply:
``getZonePluginInfoCommandPluginName(String pluginName)``  line 184